supertokens_python/recipe/passwordless/emaildelivery/service/smtp/implementation.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from typing import Any, Dict
+
+from supertokens_python.ingredients.emaildelivery.services.smtp import (
+    GetContentResult, ServiceInterface)
+from supertokens_python.recipe.passwordless.emaildelivery.service.smtp.pless_email import \
+    pless_email_content
```

Comment:
not sure why this PR has emaildelivery related files

Reply:
Removed.